src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -373,6 +383,13 @@ def convert_epa_params_to_mvs(epa_dict):
                             "this warning in data_parser.py and remove the warning and the 7 "
                             "lines of code above it as well"
                         )
+
+                if asset_group == ENERGY_CONSUMPTION:
+                    if DSM not in dict_asset[asset_label]:
+                        dict_asset[asset_label][DSM] = False
+
+                if EMISSION_FACTOR not in dict_asset[asset_label]:
+                    dict_asset[asset_label][EMISSION_FACTOR] = {VALUE: 0}
```

Comment:
Shouldn´t this be automatically be updated with the default value - or do you want to avoid the warning message?

Would it be possible to use the default value as provided in `constants.py`?

Reply:
The idea was to prevent not understanding why the EMISSION_FACTOR value is always 0 if it is implemented in the EPA in the future ;)

Using the default value is a good idea :)

Reply:
Done in 34dd06282c6da8e6395626b7c4730c2a89a3e63c